katalog/cert-manager/cert-manager-controller/rbac.yml

```diff
@@ -8,13 +8,24 @@ automountServiceAccountToken: true
 kind: ServiceAccount
 metadata:
   name: cert-manager
-  labels: {}
+  labels:
+    app: cert-manager
+    app.kubernetes.io/name: cert-manager
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/component: "controller"
+    app.kubernetes.io/version: "v1.16.1"
```

Comment:
@nutellinoit thoughts on adding these labels?

IMO is a good thing to start adding them and follow the Kubernetes recommended labels format, but I don't remember why we haven't done it before, you said that maybe was because of fluentbit.

Reply:
I think we fixed the behaviour of fluentbit in the past and it should also be working now. The problem on having app label adn app.kubernetes.something label can cause issue when evaluating labels (if no sanitization is done) and on ingesting them in opensearch, since the auto mapping will cause all the logs to be rejected. but in this case if it's a rbac there should be no impacts